suite-native/device-manager/src/components/DeviceManagerContent.tsx

```diff
@@ -63,6 +76,10 @@ export const DeviceManagerContent = () => {
         return null;
     }
 
+    // offset and the height of the content header
+    const scrollViewTopOffset = insets.top + utils.spacings.large + 86;
```

Comment:
`utils.spacings.large + 86` = `100`. Please create some const at the top of the file with descriptive name so the comment is not needed. 

e.g. const HEADER_HEIGHT = 100, or something like this.

Reply:
10e7175793